internal/platform-support/dgpu/dgpu.go

```diff
@@ -25,22 +27,79 @@ import (
 )
 
 // NewForDevice creates a discoverer for the specified Device.
+// nvsandboxutils is used for discovery if specified, otherwise NVML is used.
 func NewForDevice(d device.Device, opts ...Option) (discover.Discover, error) {
 	o := new(opts...)
+	o.isMigDevice = false
```

Comment:
nit :`isMigDevice` should already be initialized to `false` and this is unneeded.

Reply:
Removed this, thanks.